plugins/plugin-sidecar/src/view/components/LeftNavSidecar.tsx

```diff
@@ -174,7 +174,7 @@ export default class LeftNavSidecar extends BaseSidecar<NavResponse, State> {
         <KuiContent
           tab={this.state.tab}
           mode={this.state.allNavs[navIdx].tabs[tabIdx]}
-          response={this.state.response[navIdx]}
+          response={this.state.response[Object.keys(this.state.response)[navIdx]]}
```

Comment:
maybe some clarification here? this looks weird, and hard to understand.

Reply:
solved, thanks!